### ISSUE TITLE
Application crashes on startup: Linux Mint 21.3

### ISSUE THREAD

**Issue by martinprad0** [nicotine-plus/nicotine-plus#2856]
Nicotine+ version:  Nicotine+ Version: 3.3.0
GTK Version: 4.12.4
Python Version: 3.11.7 (linux)

OS: Linux Mint 21.3 x86_64 
Kernel: 5.15.0-92-generic 

### Describe the bug

When I enter the application, the following window appears:
```
Type: <class 'BufferError'>
Value: Existing exports of data: object cannot be re-sized
Traceback:   File "/app/lib/python3.11/site-packages/pynicotine/gtkgui/application.py", line 848, in _raise_exception
    raise exc_value
  File "/usr/lib/python3.11/threading.py", line 1045, in _bootstrap_inner
    self.run()
  File "/app/lib/python3.11/site-packages/pynicotine/slskproto.py", line 2685, in run
    self._process_ready_sockets(current_time)
  File "/app/lib/python3.11/site-packages/pynicotine/slskproto.py", line 2456, in _process_ready_sockets
    self._process_ready_input_socket(sock, current_time)
  File "/app/lib/python3.11/site-packages/pynicotine/slskproto.py", line 2411, in _process_ready_input_socket
    self._process_conn_incoming_messages(conn_obj_established)
  File "/app/lib/python3.11/site-packages/pynicotine/slskproto.py", line 2467, in _process_conn_incoming_messages
    self._process_server_input(conn_obj)
  File "/app/lib/python3.11/site-packages/pynicotine/slskproto.py", line 1307, in _process_server_input
    self._send_message_to_server(ServerDisconnect())
  File "/app/lib/python3.11/site-packages/pynicotine/slskproto.py", line 1518, in _send_message_to_server
    self._process_outgoing_message(message)
  File "/app/lib/python3.11/site-packages/pynicotine/slskproto.py", line 2539, in _process_outgoing_message
    process_func(msg_obj)
  File "/app/lib/python3.11/site-packages/pynicotine/slskproto.py", line 2301, in _process_internal_messages
    self._server_disconnect()
  File "/app/lib/python3.11/site-packages/pynicotine/slskproto.py", line 1479, in _server_disconnect
    self._close_connection(self._conns, sock)
  File "/app/lib/python3.11/site-packages/pynicotine/slskproto.py", line 937, in _close_connection
    conn_obj.ibuf.clear()
```

### Additional context

![image](https://github.com/nicotine-plus/nicotine-plus/assets/66460425/9f2f6c56-b4d1-4a60-b56c-f0263f71101f)

**Comment by slook** [comment]
Closing as duplicate of https://github.com/nicotine-plus/nicotine-plus/issues/2850

Fixed in Nicotine+ 3.3.1dev1 available at https://github.com/nicotine-plus/nicotine-plus/blob/master/doc/TESTING.md

**Comment by Ham23010** [comment]
Linux mint users;;;

uninstal Nicotine Flat pack and instal this;

sudo add-apt-repository ppa:nicotine-team/unstable
sudo apt update; sudo apt install nicotine

works again
succes
